Sources/Application/Game/Word.swift

```diff
@@ -0,0 +1,20 @@
+public class Word {
+	public var gameWord: String
+
+	public init(gameWord: String) {
+		self.gameWord = gameWord
+	}
+
+	public func isValid() -> Bool {
+		return gameWord != "" && hasOnlyLetters()
+	}
+
+	private func hasOnlyLetters() -> Bool {
+		for character in gameWord.characters {
+			if !((character >= "a" && character <= "z") || (character >= "A" && character <= "Z")) {
+				return false
+			}
```

Comment:
There seems to be some repetitive things between the word and the guess, maybe pulling out some sort of `CharacterValidator` is appropriate